packages/@aws-cdk/aws-cognito/lib/user-pool-email.ts

```diff
@@ -0,0 +1,267 @@
+
+/**
+ * Configuration for Cognito sending emails via Amazon SES
+ */
+export interface SESOptions {
+  /**
+   * The verified Amazon SES email address that Cognito should
+   * use to send emails.
+   *
+   * The email address used must be a verified email address
+   * in Amazon SES and must be configured to allow Cognito to
+   * send emails.
+   *
+   * @see https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-email.html
+   */
+  readonly fromEmail: string;
+
+  /**
+   * An optional name that should be used as the sender's name
+   * along with the email.
+   *
+   * @default - no name
+   */
+  readonly fromName?: string;
+
+  /**
+   * The destination to which the receiver of the email should reploy to.
+   *
+   * @default - same as the fromEmail
+   */
+  readonly replyTo?: string;
+
+  /**
+   * The name of a configuration set in Amazon SES that should
+   * be applied to emails sent via Cognito.
+   *
+   *  @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cognito-userpool-emailconfiguration.html#cfn-cognito-userpool-emailconfiguration-configurationset
+   *
+   * @default - no configuration set
+   */
+  readonly configurationSetName?: string;
+
+  /**
+   * Required if the UserPool region is different than the SES region.
+   *
+   * If sending emails with a Amazon SES verified email address,
+   * and the region that SES is configured is different than the
+   * region in which the UserPool is deployed, you must specify that
+   * region here.
+   *
+   * Must be 'us-east-1', 'us-west-2', or 'eu-west-1'
+   *
+   * @default - The same region as the Cognito UserPool
+   */
+  readonly sesRegion?: string;
+}
+
+/**
+ * Configuration settings for Cognito default email
+ */
+export interface CognitoEmailOptions {
+  /**
+   * The verified email address in Amazon SES that
+   * Cognito will use to send emails. You must have already
+   * configured Amazon SES to allow Cognito to send Emails
+   * through this address.
+   *
+   * @see https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-email.html
+   *
+   * @default - Cognito default email address will be used
+   * 'no-reply@verificationemail.com'
+   */
+  readonly fromEmail?: string;
+
+  /**
+   * The destination to which the receiver of the email should reploy to.
+   *
+   * @default - same as the fromEmail
+   */
+  readonly replyTo?: string;
+
+  /**
+   * Required if the UserPool region is different than the SES region.
+   *
+   * If sending emails with a Amazon SES verified email address,
+   * and the region that SES is configured is different than the
+   * region in which the UserPool is deployed, you must specify that
+   * region here.
+   *
+   * Must be 'us-east-1', 'us-west-2', or 'eu-west-1'
+   *
+   * @default - The same region as the Cognito UserPool
+   */
+  readonly sesRegion?: string;
+}
+
+/**
+ * Configuration for Cognito email settings
+ */
+export interface EmailConfiguration {
```

Comment:
Use `Config` as the suffix instead of `Configuration`. This is the typical CDK convention for bind results.

Reply:
Done.